src/abstract/ThresholdGovernor.sol

```diff
@@ -156,14 +174,26 @@ abstract contract ThresholdGovernor is IThresholdGovernor, BatchGovernor {
      * @param newThresholdRatio_ The new threshold ratio.
      */
     function _setThresholdRatio(uint16 newThresholdRatio_) internal {
-        if (newThresholdRatio_ > ONE || newThresholdRatio_ < _MIN_THRESHOLD_RATIO)
+        if (newThresholdRatio_ > ONE || newThresholdRatio_ < _MIN_THRESHOLD_RATIO) {
             revert InvalidThresholdRatio(newThresholdRatio_, _MIN_THRESHOLD_RATIO, ONE);
+        }
 
+        emit QuorumNumeratorUpdated(thresholdRatio, newThresholdRatio_);
         emit ThresholdRatioSet(thresholdRatio = newThresholdRatio_);
```

Comment:
Shouldn't we keep only one event?
The same information is logged twice.

Reply:
Unfortunately, the function and proposal name is `setThresholdRatio`, and is used by our front end. Meanwhile, `QuorumNumeratorUpdated` is the standard event needed to comply. This is one of the main reasons why my previous branch/PR renamed all "thresholdRatio" to "quorumNumerator", so that we would not need to satisfy two different expected nomenclatures.